src/Runner.Worker/ContainerOperationProvider.cs

```diff
@@ -395,14 +425,14 @@ private async Task RemoveContainerNetworkAsync(IExecutionContext executionContex
             }
         }
 
-        private async Task ContainerHealthcheck(IExecutionContext executionContext, ContainerInfo container)
+        private async Task<string> ContainerHealthcheck(IExecutionContext executionContext, ContainerInfo container)
```

Comment:
should we just make this `Task<bool>`? and return `true` in line 435 and return `true` in line 446 if `serviceHealth =="healthy"`

Reply:
update accordingly @TingluoHuang  , thank you!